bindings/pyroot_experimental/PyROOT/src/PyROOTModule.cxx

```diff
@@ -0,0 +1,104 @@
+
+// Bindings
+#include "PyROOTPythonize.h"
+#include "PyROOTStrings.h"
+#include "PyROOTWrapper.h"
+
+// Cppyy
+#include "CPyCppyy.h"
+#include "CallContext.h"
+#include "ProxyWrappers.h"
+#include "Utility.h"
+
+// ROOT
+#include "TROOT.h"
+#include "TSystem.h"
+
+// Standard
+#include <string>
+#include <sstream>
+#include <utility>
+#include <vector>
+
+using namespace CPyCppyy;
+
+namespace PyROOT {
+PyObject *gRootModule = 0;
+}
+
+// Methods offered by the interface
+static PyMethodDef gPyROOTMethods[] = {{(char *)"PythonizeTTree", (PyCFunction)PyROOT::PythonizeTTree, METH_VARARGS,
+                                        (char *)"Pythonizations for class TTree"},
+                                       {NULL, NULL, 0, NULL}};
+
+#if PY_VERSION_HEX >= 0x03000000
+struct module_state {
+   PyObject *error;
+};
+
+#define GETSTATE(m) ((struct module_state *)PyModule_GetState(m))
+
+static int rootmodule_traverse(PyObject *m, visitproc visit, void *arg)
+{
+   Py_VISIT(GETSTATE(m)->error);
+   return 0;
+}
+
+static int rootmodule_clear(PyObject *m)
+{
+   Py_CLEAR(GETSTATE(m)->error);
+   return 0;
+}
+
+static struct PyModuleDef moduledef = {PyModuleDef_HEAD_INIT,       "libROOTPython",  NULL,
+                                       sizeof(struct module_state), gPyROOTMethods,   NULL,
+                                       rootmodule_traverse,         rootmodule_clear, NULL};
+
+
+/// Initialization of extension module libROOTPython
+
+#define PYROOT_INIT_ERROR return NULL
+extern "C" PyObject *PyInit_libROOTPython()
+#else
```

Comment:
Can you add `// PY_VERSION_HEX >= 0x03000000` to clarify which `#if` this corresponds to? It's far, far away.